src/bindings/python/src/pyopenvino/graph/descriptors/tensor.cpp

```diff
@@ -106,6 +106,63 @@ void regclass_graph_descriptor_Tensor(py::module m) {
                 :return: Any name.
                 :rtype: string
              )");
+   
+   tensor.def("get_lower_value",
+               &ov::descriptor::Tensor::get_lower_value,
+               R"(
+                Returns the lower bound of the tensor.
+
+                :return: Lower bound.
+                :rtype: Tensor
```

Comment:
```suggestion
                :rtype: openvino.runtime.Tensor
```
Better to have it like this for proper documentation generation